Source/FormTextField.swift

```diff
@@ -86,28 +85,16 @@ public class FormTextField: UITextField, UITextFieldDelegate {
     private lazy var customClearButton: UIButton? = {
         var button: UIButton? = nil
 
-        if self.isUsingClearButton {
```

Comment:
If you don't use this bool, this doesn't have to be an optional anymore :v: But I think we can leave it an optional, to disable the use of the clearButton.